human_tests/Human_latent_transform_simulation.ipynb

```diff
@@ -190,7 +192,7 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "There are 3219 experiments in the compendium\n",
+      "There are 3220 experiments in the compendium\n",
```

Comment:
what caused this change?

Reply:
Good eye. There is a difference because one of the entries in the metadata file didn't align with the downloaded expression data and caused an error when selecting the experiment ids with associated expression data. I manually changed the metadata file here.